lib/publiccloud/gcp_client.pm

```diff
@@ -33,10 +34,15 @@ has vault => undef;
 
 sub init {
     my ($self) = @_;
-
-    $self->vault(publiccloud::vault->new());
-
-    $self->create_credentials_file();
+    # Transition period between using Vault or the micro service to get the credentials
```

Comment:
I actually don't like this sentence, but my brain is fried.. feel free to propose a more understandable statement :)

Reply:
```suggestion
    # For now we support Vault and the credentials-microservice. Vault will be removed after a certain transition period
```